content/security/rate-limiting.md

```diff
@@ -102,25 +102,29 @@ This module can work with websockets, but it requires some class extension. You
 ```typescript
 @Injectable()
 export class WsThrottlerGuard extends ThrottlerGuard {
-  async handleRequest(
-    context: ExecutionContext,
-    limit: number,
-    ttl: number,
-  ): Promise<boolean> {
+  async handleRequest(context: ExecutionContext, limit: number, ttl: number): Promise<boolean> {
     const client = context.switchToWs().getClient();
-    const ip = client.conn.remoteAddress;
+    const ip = ['_socket']
+      .map((key) => client[key])
+      .filter((obj) => obj)
+      .shift().remoteAddress;
```

Comment:
This entire map and filter is not longer needed, right? just `client._socket.remoteAddress`

Reply:
Yes